### DISCUSSION TITLE
I can‘t create file

### DISCUSSION THREAD

**Discussion by brightleo** [cline/cline#718] [Feature Requests]
I have no way to create the file and the following error occurs
------------------------------
You did not use a tool in your previous response! Please retry with a tool use.
[ERROR] You did not use a tool in your previous response! Please retry with a tool use.

# Reminder: Instructions for Tool Use
------------------------------

**Comment by elemental-mind** [comment]
Hey, this is not a feature request. Could you close this discussion and make it an issue if it's still unresolved?
